packages/mui-material/src/CssBaseline/CssBaseline.js

```diff
@@ -3,15 +3,15 @@ import PropTypes from 'prop-types';
 import useThemeProps from '../styles/useThemeProps';
 import GlobalStyles from '../GlobalStyles';
 
-export const html = (theme) => ({
+export const html = (theme, enableColorScheme) => ({
   WebkitFontSmoothing: 'antialiased', // Antialiasing.
   MozOsxFontSmoothing: 'grayscale', // Antialiasing.
   // Change from `box-sizing: content-box` so that `width`
   // is not affected by `padding` or `border`.
   boxSizing: 'border-box',
   // Fix font resize problem in iOS
   WebkitTextSizeAdjust: '100%',
-  colorScheme: theme.palette.mode,
+  ...(enableColorScheme && { colorScheme: theme.palette.mode }),
 });
```

Comment:
I assume these functions are exported to be used by `ScopedCssBaseline`, but not used by dev. I do not found any use of them somewhere else. That's why I modified their signature.